content/concepts/configuration.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Configuration
+---
+
+## Overview 
```

Comment:
Same as @bebraw already mentioned. This reads more like an introduction than an overview.